TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -85,7 +123,8 @@ void ticker_callback_2_switch_to_1(void) {
     ticker_callback_2_led();
 }
 
-void wait_and_print() {
+void wait_and_print()
```

Comment:
This function is not used, would it be possible to remove it as well as symbols only used by it ?

Reply:
removed